modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -3,46 +3,48 @@
 :page-aliases: cluster-administration:cluster-balancing.adoc
 :page-categories: Management
 
-When a topic is created, Redpanda evenly distributes its partitions by sequentially allocating them to the cluster broker with the least number of partitions. For existing topics, Redpanda automatically provides leadership balancing and partition rebalancing when brokers are added or decommissioned.
+Cluster balancing is crucial for optimal performance. Unbalanced clusters can saturate resources on one or more brokers, impacting  throughput and latency. Furthermore, a cluster with replicas on a down broker risks availability loss if more brokers fail, and a cluster that keeps losing brokers without healing eventually risks data loss. Redpanda provides various tools to balance clusters for best performance. 
 
-With an Enterprise license, you can additionally enable _Continuous Data Balancing_ to continuously monitor broker and rack availability and disk usage. This enables self-healing clusters that dynamically balance partitions. It also continuously maintains adherence to rack-aware replica placement policy and self-heals after rack (or availability zone) failure or replacement. See xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Configure Continuous Data Balancing].
-
-Cluster balancing protects you from unbalanced systems that saturate resources on one or more brokers. This can affect throughput and latency. Furthermore, a cluster with replicas on a down broker risks availability loss if more brokers fail, and a cluster that keeps losing brokers without healing eventually risks data loss.
-
-The following table summarizes the various balancing options in Redpanda:
-
-
-.Cluster balancing options
-[cols="4,2,11"]
+[.no-clip]
 |===
-|Balancer |xref:get-started:licenses.adoc[License] |Description
+| Balancer |Description
 
-|Partition leadership balancing
-|Community Edition, Enterprise Edition
-|Topic-aware balancer designed to distribute partition leadership across available nodes. This helps avoid topic leadership hotspots on one or a few specific nodes in your cluster.
+| <<Partition leadership balancing>>
+| Topic-aware balancing that transfers the leadership of a broker's partitions to other replicas. This helps avoid topic leadership hotspots on one or a few specific brokers in your cluster. 
+ +
+ +
+Redpanda automatically provides leadership balancing when brokers are added or decommissioned.
```

Comment:
Wait, what?  @micheleRP leadership balancing happens _continuously_ based on a heartbeat, not only when brokers are added/removed https://docs.redpanda.com/current/manage/cluster-maintenance/cluster-balancing/#partition-leadership-balancing